### PR TITLE
net-misc/ptpd: update package and resolve bug #587328

### PR DESCRIPTION
**Pull request by j123b567**
Project moved from sourceforge to github so first commit just updates original ebuild to use new source.
Second commit solves bug #587328

### CODE REVIEW

**Comment by j123b567** [comment]
Btw, I'm the proxy-maintainer of this package. Is there a way how to name my github account in metadata.xml?

**Comment by monsieurp** [comment]
Yes, here's an example: https://gitweb.gentoo.org/repo/gentoo.git/tree/app-text/jist/metadata.xml#n9

File: net-misc/ptpd/ptpd-2.3.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+	newinitd "${FILESDIR}/ptpd2.rc" ptpd2
+	newconfd "${FILESDIR}/ptpd2.confd" ptpd2
+
+	systemd_dounit "${FILESDIR}/ptpd2.service"
+}
+
+pkg_postinst() {
+	elog "Do not forget to setup correct network interface."
+	elog "Change the config file /etc/ptpd2.conf to suit your needs."
```
Comment (gktrk) [inline net-misc/ptpd/ptpd-2.3.1-r1.ebuild]: This should be `${EROOT}etc/ptpd2.conf`

File: net-misc/ptpd/ptpd-2.3.1-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils flag-o-matic systemd
```
Comment (gktrk) [inline net-misc/ptpd/ptpd-2.3.1-r1.ebuild]: inherit on eutils seems redundant

**Comment by j123b567** [comment]
All comments should be fixed now.

**Pull request closed without merging**